xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/client/common/LedgerIndex.java

```diff
@@ -34,9 +34,17 @@ public LedgerIndex(String value) {
    * @param value A {@link String} containing either an integer or a shortcut.
    *
    * @return A {@link LedgerIndex} with the given value.
+   *
+   * @throws NumberFormatException if value is an invalid index
    */
-  public static LedgerIndex of(String value) {
-    return new LedgerIndex(value);
+  public static LedgerIndex of(String value)
```

Comment:
Let's add some unit test coverage for the three text-values, and probably for a `null` value too?

Reply:
I added some unit tests as requested.

I'd also like to point out that the `LedgeIndexTest.createInvalidNumericalLedgerIndex` could be removed because it does not make sense to test the `LedgerIndex.plus` method after bad calls to `LedgerIndex.of`.

Reply:
Yes, I agree -- it looks like your new test (see [here](https://github.com/XRPLF/xrpl4j/pull/98/files#diff-ba78bf512011908df564830de4f68373ea102c5f6ec9b3dea7a1febef3936615R31)) covers everything that was covered by `createInvalidNumericalLedgerIndex` -- feel free to remove it and verify that test coverage is the same?